Source/ThirdParty/Assimp/CMakeLists.txt

```diff
@@ -106,6 +106,28 @@ INCLUDE_DIRECTORIES(
 
 LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake-modules" )
 
+# Grouped compiler settings
+IF ((CMAKE_C_COMPILER_ID MATCHES "GNU") AND NOT MINGW)
+  # hide all not-exported symbols
+  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fvisibility=hidden -fPIC -Wall -std=c++0x")
+  SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
+  SET(LIBSTDC++_LIBRARIES -lstdc++)
+ELSEIF(MSVC)
+  # enable multi-core compilation with MSVC
+  add_compile_options(/MP)
+
+  # disable "elements of array '' will be default initialized" warning on MSVC2013
+  IF(MSVC12)
```

Comment:
https://cmake.org/cmake/help/latest/variable/MSVC12.html

Reply:
I restored the section "Grouped compiler settings" from the original file from v4.1.0 of the assimp library with some minor fixes. I will remove the rest of the conditions and leave this one for MINGW.

# Grouped compiler settings
IF ((CMAKE_C_COMPILER_ID MATCHES "GNU") AND NOT CMAKE_COMPILER_IS_MINGW)
  # hide all not-exported symbols
  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fvisibility=hidden -fPIC -Wall -std=c++0x")
  SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
  SET(LIBSTDC++_LIBRARIES -lstdc++)
ELSEIF(MSVC)
  # enable multi-core compilation with MSVC
  add_compile_options(/MP)

  # disable "elements of array '' will be default initialized" warning on MSVC2013
  IF(MSVC12)
    add_compile_options(/wd4351)
  ENDIF()
ELSEIF ( "${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang" )
  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fvisibility=hidden -fPIC -Wall -Wno-long-long -std=c++11" )
  SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
ELSEIF( CMAKE_COMPILER_IS_MINGW )
  SET( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fvisibility=hidden -Wall -Wno-long-long -std=c++11" )
  SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
  ADD_DEFINITIONS( -U__STRICT_ANSI__ )
ENDIF()